### DISCUSSION TITLE
Custom TimeSpan Input Component in Fluent UI: Seeking Solutions and Customization Options

### DISCUSSION THREAD

**Discussion by evon-thomasgrabner** [microsoft/fluentui-blazor#2354] [Q&A]
Hi everyone,

I'm currently working on a project that requires an input component for **System.TimeSpan** values, formatted as hh:mm:ss. I'm looking for something similar to **FluentTimePicker**. However, I'm facing a couple of challenges:

- Limitation with FluentTimePicker: The default culture setting enforces the AM/PM format, which isn't suitable for my needs.
- Customization Needs: I need to customize the FluentTimePicker to:
    - Remove the AM/PM option.
    - Adjust the format to display and input time as hh:mm:ss.

Is there an existing Fluent UI component that caters to this requirement? If not, has anyone successfully customized the FluentTimePicker to achieve this? Any insights, code snippets, or guidance would be greatly appreciated!

Thanks in advance!

**Comment by vnbaaij** [comment]
TimePicker is using the native browser component for its display
Unless you can target specific parts with custom CSS, there is not much that can be changed, I'm afraid.
We have no other components in our collection to help you with that.
